whylog/config/__init__.py

```diff
@@ -103,6 +114,46 @@ def _filter_rule_set(self, parsers_list):
     def _get_locations_for_logs(self, logs_types_list):
         pass
 
+    def is_free_parser_name(self, parser_name):
+        return parser_name not in self._parsers
+
+    def _is_unique_parser_name(self, parser_name, black_list):
+        return self.is_free_parser_name(parser_name) and (parser_name not in black_list)
+
+    def propose_parser_name(self, line, regex_str, black_list):
+        building_words = AbstractConfig._get_building_words(line, regex_str)
+        if not building_words:
+            return uuid.uuid4()
+        if len(building_words) == 1:
+            if self._is_unique_parser_name(building_words[0], black_list):
+                return building_words[0]
```

Comment:
Shouldn't we return 3 words instead of one? One word could say not much. Better is "data_loss" instead of "data". And then is lower likelihood that names will repeat. I was thinking about 4 words - it would say much more, but it would can be too long.